infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftWorkspaceServiceAccount.java

```diff
@@ -69,6 +66,11 @@ void prepare() throws InfrastructureException {
       createExecRole(osClient, execRoleName);
     }
 
+    String viewRoleName = "view";
+    if (osClient.roles().inNamespace(projectName).withName(execRoleName).get() == null) {
```

Comment:
`execRoleName` must be changed to viewRoleName, right?

Reply:
Right. Fixed.